Roman Androsiuk/0/start.rb

```diff
@@ -0,0 +1,205 @@
+require 'roo'
+require 'roo-xls'
+# require 'rubocop'
+xlfilenow = Roo::Excelx.new('./data/Average_prices(serv)-10-2018.xlsx') # Info about prices in Minsk these days
```

Comment:
Put code that is actually executed at the end of the file. Also you can put it all in main method and call it at the end of a file.